src/frontend/screens/Game/GamePage/components/ReleaseDate.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import GameContext from '../../GameContext'
+
+type ReleaseDateProps = {
+  date: string[] | undefined
+}
+
+// convert date to current locale using Intl module
+function convertDate(date: string) {
+  // Extract only the date part if the string contains extra information
+  const match = date.match(/(\w+ \d{1,2}, \d{4})/)
+  if (match) {
+    date = match[1]
+  }
+
+  const options: Intl.DateTimeFormatOptions = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric'
+  }
+
+  const dateObj = new Date(date)
+
+  // Check if dateObj is a valid date
+  if (isNaN(dateObj.getTime())) {
+    return null
+  }
+
+  return dateObj.toLocaleDateString(undefined, options)
```

Comment:
we should probably use the user's selected language here instead of undefined, now it uses the system's language instead of heroic's language

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/ed9a323e-d7ad-47fc-a42d-d12d84b46007)

Reply:
I was thinking about that and I think the system language is ideal in this case.
Because imagine if the person, like me, is from Brazil, latin America in general. And then uses Heroic in English. We will see the date in US standards and that is not ideal IMO. 
Also because the language is different from metrics.

imagine if we decide to show measurements in inches and foot instead of meters and centimeters just because we want heroic translations to be in English 😓

Reply:
Also, it is the same logic we are applying on the time container to show the information of last played, etc.

Reply:
yeah, honestly I don't know what users expect, I'm fine with this